File: LeetCode/Prep/Top_interview_questions_easy/Arrays/RemoveDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=0
        j=0
        n= len(nums)
        while j<n:
            if nums[i] == nums[j]:
                j+=1
            else:
                nums[i+1] = nums[j]
                i+=1
        
        return i+1

File: LeetCode/Prep/Top_interview_questions_easy/Arrays/RemoveDuplicatesFromSortedArray.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0;
        int j=0;
        int n = nums.length;
         while(j<n){
            if(nums[i]== nums[j])
                 j++;
            else{
                nums[i+1] = nums[j];
                i++;
            }
             
         }
        return i+1;
    }
}
